README.md

```diff
@@ -1,46 +1,84 @@
-# Risitas_BOT
+<h1 align="center">🤖 Risitas Discord Bot 🤖</h1>
+<p>
+  <img alt="Version" src="https://img.shields.io/badge/version-1.0-blue.svg?cacheSeconds=2592000" />
+  <a href="#" target="_blank">
+    <img alt="License: MIT" src="https://img.shields.io/badge/License-MIT-yellow.svg" />
+  </a>
+  <img alt="Github Stars" src="https://img.shields.io/github/stars/Floriansylvain/Risitas_BOT?style=social">
```

Comment:
I added the github stars counter but you can delete it for the moment if you want

Reply:
What an outstanding work, thank you !